apptest.go

```diff
@@ -449,6 +454,46 @@ func (a *AppSetup) createApps(ctx context.Context, apps []App) error {
 	return nil
 }
 
+func (a *AppSetup) createCatalogs(ctx context.Context, apps []App) error {
+	for _, app := range apps {
+		catalogURL, err := getCatalogURL(app)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		a.logger.Debugf(ctx, "creating %#q catalog cr", app.CatalogName)
+
+		catalogCR := &v1alpha1.Catalog{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      app.CatalogName,
+				Namespace: metav1.NamespaceDefault,
```

Comment:
To make it easier, I would like to give `default` namespace for start. We can elaborate a bit more later.

Reply:
This is fine. We'll use the default namespace for public catalogs.